src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -54,6 +54,14 @@ impl HtmlHandlebars {
                                                                            to str")
                                                        })?;
 
+                // "print.html" is used for the print page.
+                if path == "print.md" {
```

Comment:
What happens if the path is `./print.md` instead? It might be better to do `if ch.path == Path::new("print.md")` to avoid those sorts of ambiguities.